sdk/forms/src/main/java/com/klaviyo/forms/bridge/CompanyObserver.kt

```diff
@@ -11,11 +11,6 @@ import com.klaviyo.forms.reInitializeInAppForms
  * to reload klaviyo.js for the new company
  */
 internal class CompanyObserver : JsBridgeObserver {
-    /**
-     * At this time, company ID doesn't have a handshake spec because
-     * it only resets the webview, doesn't communicate with the onsite module
-     */
-    override val handshake: HandshakeSpec? = null
```

Comment:
Realized that these handshake items make more sense to live in the `JsBridge` interface, similar to how native bridge has a handshake collection, than on the individual observers. Not all observers even have a handshake required.